scripts/text_classification/dataloader.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
```

Comment:
Could you add license headers to the files, e.g. https://github.com/dmlc/gluon-nlp/blob/master/src/gluonnlp/embedding/__init__.py#L1-L19 ?